layer3/MoleculeExporter.cpp

```diff
@@ -1820,7 +1820,7 @@ struct MoleculeExporterChemPy : public MoleculeExporter {
 
   void writeAtom() override {
     PyObject *atom = CoordSetAtomToChemPyAtom(G,
-        m_iter.getAtomInfo(), m_coord, getRefPtr(),
+        m_iter.getAtomInfo(), m_last_obj, m_coord, getRefPtr(),
```

Comment:
I think `m_iter.obj` is meant to be used instead. `m_last_obj` I think is mostly just used to distinguish object separation from adjacent atoms being iterated.